Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:20.04 AS build
+
+ARG VERSION=1.9.0
```

Comment:
can we change to `1.10.0` now for the issue you created?

Reply:
sure